src/markdownwidget/plugin.ts

```diff
@@ -13,6 +13,8 @@ import {
   MarkdownWidgetFactory
 } from './widget';
 
+import './theme.css';
```

Comment:
I'm not sure if this is the correct way to do things.  It was the only way I could get the CSS to take effect.